mmv1/third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -6321,3 +6350,35 @@ resource "google_compute_instance" "foobar" {
 `, instance)
 }
 
+func testAccComputeInstance_metadataStartupScript(instance, machineType, metadata string) string {
```

Comment:
I saw some discussion on this in the GitHub issue, but do we actually need to change `machineType` to trigger the problem? If it is unrelated, we may want to remove it from the test.

Reply:
Yes, that forces the vm to stop & restart. The issue happens only when the instance restart